src/screens/flows/Send/components/ReviewSend/styles.ts

```diff
@@ -42,4 +42,9 @@ export default StyleSheet.create({
   subtitle: {
     ...fontMaker({ size: 16, weight: MEDIUM, color: Colors.White.Secondary }),
   },
```

Comment:
Should you change this?
```suggestion
  subtitle: fontMaker({ size: 16, weight: MEDIUM, color: Colors.White.Secondary }),
```